wolfssl/wolfcrypt/types.h

```diff
@@ -556,6 +556,8 @@ typedef struct w64wrapper {
                 #endif
                 #define XREALLOC(p, n, h, t) wolfSSL_Realloc((p), (n), (h), (t))
             #endif /* WOLFSSL_DEBUG_MEMORY */
+        #elif  defined(WOLFSSL_EMBOS) && !defined(XMALLOC_USER) && !defined(NO_WOLFSSL_MEMORY) && !defined(WOLFSSL_STATIC_MEMORY)
```

Comment:
Our style is to have 80 characters per line.  even though this isn't followed in the code surrounding this change (ie XFREE above), but that is for readability.  Please use `\` to keep lines below 80 characters.